### PR TITLE
Attempting to improve flakiness of async timing tests

### PR DESCRIPTION
**Pull request by corey-nm**
I'm unsure if this addresses the issue on github workflows

### CODE REVIEW

File: tests/deepsparse/pipelines/test_pipeline.py [hunk 1]
```diff
@@ -114,30 +114,34 @@ def test_pipeline_executor_num_workers():
...
-    # in each word. So "aaaa" will sleep for 50ms whereas "a" will sleep for 10ms
+    # attempts to verify that pipeline calls to engine are async
+    # by executing at timing test with different number of workers
+
     executor = ThreadPoolExecutor(max_workers=1)
     pipeline = Pipeline.create("token_classification", batch_size=1, executor=executor)
 
-    def sleep_for_10ms(xs):
-        time.sleep(10 / 1000)
+    def sleepy_engine(xs):
```
Comment (bfineran) [inline tests/deepsparse/pipelines/test_pipeline.py:123]: nit: name

File: tests/deepsparse/pipelines/test_pipeline.py [hunk 2]
```diff
@@ -114,30 +114,34 @@ def test_pipeline_executor_num_workers():
...
+        # each call to engine_forward also sleeps
+        time.sleep(20 / 1000)
         return pipeline.engine(xs)
```
Comment (bfineran) [inline tests/deepsparse/pipelines/test_pipeline.py:126]: just noticed this, intentional? issue with mocking?
```suggestion
        return pipeline.engine_forward(xs)
```

Comment (corey-nm) [reply tests/deepsparse/pipelines/test_pipeline.py:126]: Intentional - this sleep_engine function replaces engine_forward

File: tests/deepsparse/pipelines/test_pipeline.py (hunk 1 above)
Comment (corey-nm) [reply tests/deepsparse/pipelines/test_pipeline.py:123]: What would you prefer?

**Review by bfineran — APPROVED** [review]

**Review by rahul-tuli — APPROVED** [review]

**Pull request merged**